crates/bevy_ecs/src/hierarchy.rs

```diff
@@ -22,7 +22,9 @@ use core::slice;
 use disqualified::ShortName;
 use log::warn;
 
-/// A [`Relationship`](crate::relationship::Relationship) component that creates the canonical
+/// Stores the parent entity of the child entity with this component.
```

Comment:
```suggestion
/// Stores the parent entity of this child entity with this component.
```

Or even something like:
```
/// This [`Component`] makes this entity a child of some parent entity by holding the parent's unique [`Entity`].
```

I'm no expert but I think it would be helpful to add something like the above to further clarify that this component doesn't just link a child to a parent. It also *makes* *this* particular entity the child of another.

I think its also helpful to include the words Child Of in the explanation. Otherwise my first thought would be, "Well then why not just call it Parent?" 

Just a thought.